test/bitcoingold.test.js

```diff
@@ -1,25 +1,28 @@
 /* global describe, it */
 
 var assert = require('assert')
-var bscript = require('../src/script')
-var bcrypto = require('../src/crypto')
-var ECPair = require('../src/ecpair')
-var NETWORKS = require('../src/networks')
-var TransactionBuilder = require('../src/transaction_builder')
-var Transaction = require('../src/transaction')
-
-describe('TransactionBuilder', function () {
-  var network = NETWORKS['bitcoingold']
-  it('goldtestcase', function () {
+
+const {
+  Transaction,
+  TransactionBuilder,
+  networks,
+  ECPair,
+  script,
+  crypto,
+} = require('..')
```

Comment:
Probably missing something, but we're in `test` and all imports used to be at `../src/`, why would we be getting them from `..` now? Aren't you getting the test files now, instead?